bitcoin/src/blockdata/transaction.rs

```diff
@@ -678,6 +678,9 @@ impl cmp::Ord for Transaction {
 }
 
 impl Transaction {
+    /// The maximum weight for transactions we're willing to relay/mine.
+    pub const MAX_STANDARD_WEIGHT: Weight = Weight::from_wu(400_000);
```

Comment:
Can we have a link to the line of code in Core in the docs here please so devs can quickly jump to it in future. Also the docs shouldn't be the same as Core's because `rust-bitcoin` is not Core, "we" don't relay or mine.

Reply:
As these rules are subject to change, ideally we want also want to say "as of bitcoin core 25.0" or something similar.

Reply:
> Can we have a link to the line of code in Core in the docs here please so devs can quickly jump to it in future. 

Sure no problem.  I try to find the explanation using git blame but pasting the link directly in the code works too.

> Also the docs shouldn't be the same as Core's because rust-bitcoin is not Core, "we" don't relay or mine.

> As these rules are subject to change, ideally we want also want to say "as of bitcoin core 25.0" or something similar.

Updated.